src/components/LoadingIndicator/LoadingIndicator.jsx

```diff
@@ -79,7 +89,7 @@ const LoadingIndicator = createClass({
 					transitionLeaveTimeout={300}
 				>
 					{isLoading && (
-						<div className={cx('&-message-container', { '&-has-overlay': hasOverlay })}>
+						<div className={cx('&-message-container', { [`&-has-overlay-${overlayKind}`]: hasOverlay })}>
```

Comment:
Yep, you're right. We try to be thoughtful about our classname changes, but I don't think anyone is relying on this one. I'd mark it as a `semver-minor`.